Source/Core/Gen5/Generators/IDGenerator5.cpp

```diff
@@ -50,19 +49,13 @@ std::vector<IDState5> IDGenerator5::generate(u64 seed, u32 pid, bool checkPID)
 
         if (filter.compare(state))
         {
-            if (checkPID)
-            {
-                bool idBit = (tid & 1) ^ (sid & 1);
+            bool idBit = (tid & 1) ^ (sid & 1);
 
-                // Check if PID will be modified by the tid/sid combo
-                u16 actualPSV = (idBit ^ pidBit) ? xorPSV : psv;
-                if ((actualPSV >> 3) == state.getTSV())
-                {
-                    state.setSeed(seed);
-                    states.emplace_back(state);
-                }
-            }
-            else
+            // Check if PID will be modified by TID/SID combo if XOR box is checked
+            bool isShinyXor = checkXOR && !(idBit ^ pidBit) && (psv >> 3) == state.getTSV();
```

Comment:
What is `!(idBit ^ pidBit)` checking for. Please change it to what was described here https://github.com/Admiral-Fish/PokeFinder/pull/191#commitcomment-57075211

Reply:
> What is `!(idBit ^ pidBit)` checking for. Please change it to what was described here [#191 (comment)](https://github.com/Admiral-Fish/PokeFinder/pull/191#commitcomment-57075211)

It avoids showing IDs combo that will make the PID ^ 80000000